pyproject.toml

```diff
@@ -10,6 +10,8 @@ repository = "https://github.com/iMicknl/python-overkiz-api"
 packages = [
     { include = "pyoverkiz" }
 ]
+include = ["pyoverkiz/py.typed"]
```

Comment:
https://jugmac00.github.io/blog/bite-my-shiny-type-annotated-library/#poetry

Seems we don't need this.

Reply:
Indeed, I just run locally `poetry build` and check the archive, the file is well present.